policies: add Conflict of Interest policy (#40)
NixOS/foundation: policies/conflict_of_interest.md

# Conflict of Interest Rule

> Purpose: This conflict of interest rule aims to ensure that the foundation leading the NixOS project operates in an impartial and transparent manner, avoiding any conflicts of interest that may arise due to members' involvement in companies that benefit from the project.

Reviewer: The menion of "NixOS project" makes it sound NixOS-specific. E.g. a company might not benefit from the NixOS project, but it might profit from the Nix project.

Should this maybe be 

```suggestion
Purpose: This conflict of interest rule aims to ensure that the foundation operates in an impartial and transparent manner, avoiding any conflicts of interest that may arise due to members' involvement in companies that benefit from the ecosystem.
```

Reviewer 2: I like this and fully agree, thanks for calling that out. Do we want to add the "Nix" before ecosystem?

Reviewer: Done.

> Scope: This rule applies to all members of the NixOS Foundation.

Reviewer: Who is a member? Is it people on the board? How about people paid by the fundation?

E.g. [this document](file:///home/asymmetric/Downloads/CONFLICT_OF_INTEREST_POLICY_FINAL.pdf) specifies more clearly whom the policy applies to:

> The following are considered Covered Persons for the purposes of this Policy who have an Interest in a
Transaction:
> 1. Each member of the Board of Directors of the Organization (“Board”).
> 2. Each officer of the Organization
> 3. Each key employee shall mean employees designated by the Board as a “Key Employee”

Reviewer 2: I agree that this should be specified. And the term "key" employee I believe gives enough leeway to hire folks from consulting companies.

Reviewer 3: Managing the NGI grants I'm also affected.

> Definition: For the purposes of this rule, a conflict of interest is any situation in which a member's personal, financial, or other interests conflict, or appear to conflict, with the interests of the foundation leading the NixOS project.

Reviewer: What I like about the OSI document is that it explicitly mentions compensation/gifts, which I think is not as clearly included here.

For example: board member X gets fancy accommodation at the next NixCon paid by company Y (not their employer). This might be seen as a way to influence that board member, and should IMO be disclosed, as long as the gift is not trivial.

> Compensation includes direct and indirect remuneration as well as gifts or favors that are not insubstantial
(travel reimbursements are not considered compensation

Reviewer 4: Listing at least one example sound reasonable to me.

Guidelines:

> 1. Disclosure of Conflicts of Interest: All members of the foundation leading the open source project must disclose any conflicts of interest to the foundation's board of directors and the project's governance body as soon as they become aware of them.

Reviewer 4: Who is the project's governance body, board observers, the moderation team, contributors? Would be nice if we can clearly state this.

Can we also change the phrasing here so it says that disclosing the reason for a Conflict of Interest is highly encouraged and appreciated but it is fine to not disclose the reason for having a CoI due to NDA's or similar contracts.

Reviewer 2: **Examples of Potential Conflicts of Interest**

Board Member Employment or Income: A board member who also holds a position in a company or direct compensation tied to an entity.

Customer Relationships: If a board member holds a substantial position within a company, and one of its customers is directly involved in a topic or decision being considered by the NixOS Foundation, this could constitute a conflict of interest.

Personal Investments: A key employee has personal investment in a private company that seeks to partner with the NixOS Foundation for a project. (This one is a bit iffy so I added private since it'd be weird to disclose that I have some Amazon stocks as it has 0 effect on my decision)

Family Involvement: A foundation member whose spouse or close relative works for a vendor that supplies services to the NixOS Foundation. 

Examples of Situations Likely Not Considered Conflicts of Interest
Receiving Vendor Swag: A member receives promotional items like t-shirts or stickers from a vendor. Such low-value items generally do not constitute a conflict of interest.

Casual Business Contacts: Having a casual or non-exclusive business relationship with a company that also works with the NixOS Foundation, where no direct influence on decisions or financial benefit to the member exists.

Attendance at Industry Events: A board member attends a conference sponsored by a company that is a vendor to the NixOS Foundation, where the member does not have any decision-making authority over purchases related to the event's sponsors.

Official Foundation Representation: A board member attends meetings or events with a vendor specifically as part of their official duties for the NixOS Foundation. This involves relationship building or partnership discussions where the board member is clearly acting in their capacity as a representative of the Foundation, not for personal or external business interests.

> 2. Recusal: Members with a conflict of interest must recuse themselves from any decision-making process or vote related to the project where the conflict of interest exists. The recusal should be documented in the project's meeting minutes.

Reviewer 4: is there any reason not to upgrade the should to a must be documented?

Author: This could cause reverse problems, where everybody got some swag from company X, and so we all have to recuse ourselves from deciding on something involving that company.

The disclosure is more important than the recusal, as it allows other members and the public to place extra scrutiny on the decision.

Recusals can be reserved when the interests are too strong, or the decision is very important.

3. Transparency: The foundation leading the open source project must disclose any conflicts of interest to the project's community. Members should avoid giving the impression that their personal interests influence the foundation's decision-making process.

4. Mitigation: Members with a conflict of interest may participate in discussions related to the project but must clearly disclose their conflicts of interest and refrain from advocating for their personal interests.

> 5. Consequences: Failure to comply with this conflict of interest rule may result in disciplinary action, including but not limited to removal from the foundation's board of directors or governance body.

Reviewer: Who decides:

- what constitutes is a violation
- what the consequences are?

In the document I linked above, it is the board itself, which has obvious failure modes. They do also mention "outside experts", which could be a mitigation against those failure modes.

Reviewer 4: ```suggestion
5. Consequences: Failure to comply with this conflict of interest rule will result in disciplinary action, including but not limited to removal from the foundation's board of directors or governance body.
```
what is the point of having this policy if it could be chosen to not do anything about violations?

# Conflict of Interest Rule

Purpose: This conflict of interest rule aims to ensure that the foundation operates in an impartial and transparent manner, avoiding any conflicts of interest that may arise due to members' involvement in companies that benefit from the Nix ecosystem.

Members in this document apply to:
- Each member of the NixOS Foundation, as filled with the Kamer van Koophandel
- Each NixOS Foundation board observer
- Each key employee shall mean employees designated by the Board as a “Key Employee”  

Definition: For the purposes of this rule, a conflict of interest is any situation in which a member's personal, financial, or other interests conflict, or appear to conflict, with the interests of the foundation leading the NixOS project.

Guidelines:

> 1. Disclosure of Conflicts of Interest: All members must disclose any conflicts of interest to the foundation's board of directors and the project's governance body as soon as they become aware of them. Compensation includes direct and indirect remuneration as well as gifts or favors that are not insubstantial (travel reimbursements are not considered compensation).

Reviewer 5: ```suggestion
1. Disclosure of Conflicts of Interest: All members must disclose any conflicts of interest to the foundation's board of directors as soon as they become aware of them. Compensation includes direct and indirect remuneration as well as gifts or favors that are not insubstantial (travel reimbursements are not considered compensation).
```
Remove reference to "governance body" which doesn't exist at the moment.

2. Recusal: Members with a conflict of interest must recuse themselves from any decision-making process or vote related to the project where the conflict of interest exists. The recusal should be documented in the project's meeting minutes.

3. Transparency: Members must disclose any conflicts of interest to the project's community. Members should avoid giving the impression that their personal interests influence the foundation's decision-making process.

4. Mitigation: Members with a conflict of interest may participate in discussions related to the project but must clearly disclose their conflicts of interest and refrain from advocating for their personal interests.

> 5. Consequences: Failure to comply with this conflict of interest rule will result in disciplinary action, including but not limited to removal from the foundation's board of directors or governance body.

Reviewer 5: ```suggestion
5. Consequences: Failure to comply with this conflict of interest rule will result in disciplinary action, including but not limited to removal from the foundation's board of directors.
```
